src/defs.h

```diff
@@ -157,6 +157,14 @@ struct ref_block_list {
 
 typedef struct ref_block_list ref_block_list_t;
 
+typedef struct insn insn_t;
+
+typedef struct use_chain_node {
+    insn_t *insn;
+    struct use_chain_node *next;
+    struct use_chain_node *prev;
+} user_t;
```

Comment:
Why not naming it `use_chain_t`?

Reply:
OK, I will make the modifications. I originally wanted to emphasize that this structure is for single node.